README.md

```diff
@@ -1,2 +1,54 @@
 # ex_ngrok
-To simplify development of Elixir applications that expect callbacks from external services, this application wraps Ngrok and will start/stop Ngrok on the command line when you start/stop your Elixir application
+
+A wrapper around [Ngrok](https://ngrok.com/) providing a secure tunnel to localhost for demoing your Elixir/Phoenix web application or testing webhook integrations.
+
+Once installed, ex_ngrok will manage starting and stopping Ngrok with your application and expose Ngrok's settings to your application.
+
+## Dependencies
+
+- **[Ngrok](https://ngrok.com/) 2.x** Install it on OSX with:
+
+        $ brew cask install ngrok
+
+## Installation
+
+Add ex_ngrok to your `mix.exs` dependencies:
+
+```elixir
+def deps do
+  [{:ex_ngrok, github: "joshuafleck/ex_ngrok", only: [:dev]}]
+end
+
+def application do
+  [ applications: [:ex_ngrok] ]
```

Comment:
I might need to spend some time to test this, but I'm afraid combination of this line and suggested `deps` might cause some problems:

In deps, the option `:only` indicates to _load this dependency_ in `dev` env, but `:applications` will make it try to start it every time, in any env.

Possible approaches might be to remove the `:only` option from `deps`, or amend `:applications` to start it in the specific envs, something like this should work:

```elixir
def application do
  [ applications: env_specific_applications(Mix.env) ]
end

def env_specific_applications(:dev) do
  [:ex_ngrok]
end

def env_specific_applications(_) do
  []
end
```

Again, I might need to spend some time playing with this to finally confirm, but this is my understanding at the moment.